ios/PolyPodApp/FeatureContainer/FeatureContainerView.swift

```diff
@@ -87,17 +147,25 @@ class FeatureWebView: WKWebView {
         let configuration = WKWebViewConfiguration()
         configuration.userContentController = contentController
 
+        let scheme = PolyNav.fsPrefix.replacingOccurrences(of: "://", with: "")
```

Comment:
This code is repeated from above. Would there be a way of calling this "getscheme" or PolyNav.getScheme or something like that?